<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeWidget" name="event_tree">
      <property name="itemsExpandable">
       <bool>false</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>80</number>
      </attribute>
      <attribute name="headerMinimumSectionSize">
       <number>1</number>
      </attribute>
      <column>
       <property name="text">
        <string>Artist</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Location</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>City</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Country</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionUpdateLibrary"/>
    <addaction name="actionUpdateEvents"/>
    <addaction name="actionAddLastFm"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdateLibrary">
   <property name="text">
    <string>Update Library</string>
   </property>
  </action>
  <action name="actionUpdateEvents">
   <property name="text">
    <string>Update Events</string>
   </property>
  </action>
  <action name="actionAddLastFm">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add to Last.fm</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
